.github/workflows/testing.yml

```diff
@@ -65,51 +67,30 @@ jobs:
-      - if: ${{ (steps.cache-ecephys-datasets.outputs.cache-hit == false || steps.cache-ophys-datasets.outputs.cache-hit == false || steps.cache-behavior-datasets.outputs.cache-hit == false) && matrix.os == 'windows-latest' }}
-        name: Get git-annex - Windows
-        uses: crazy-max/ghaction-chocolatey@v1.6.0
-        with:
-          args: install git-annex --ignore-checksums
-      - if: ${{ (steps.cache-ecephys-datasets.outputs.cache-hit == false || steps.cache-ophys-datasets.outputs.cache-hit == false || steps.cache-behavior-datasets.outputs.cache-hit == false) && matrix.os == 'windows-latest' }}
-        name: Get datalad - Windows
-        run: pip install datalad==0.16.3
+          key: behavior-datasets-2022-08-18-${{ matrix.os }}-${{ steps.behavior.outputs.HASH_behavior_DATASET }}
+
 
-      - if: steps.cache-ecephys-datasets.outputs.cache-hit == false && (matrix.os == 'ubuntu-latest' || matrix.os == 'windows-latest')
-        name: "Force GIN: ecephys download"
+
+      - if: steps.cache-ephys-datasets.outputs.cache-hit == false || steps.cache-ophys-datasets.outputs.cache-hit == false || steps.cache-behavior-datasets.outputs.cache-hit == false
+        name: Install and configure AWS CLI
         run: |
-          datalad install https://gin.g-node.org/NeuralEnsemble/ephy_testing_data
-          cd ephy_testing_data
-          datalad get -r ./neuralynx/Cheetah_v5.4.0/original_data/
-          datalad get -r ./neuralynx/Cheetah_v5.5.1/original_data/
-          datalad get -r ./neuralynx/Cheetah_v5.6.3/original_data/
-          datalad get -r ./neuralynx/Cheetah_v5.7.4/original_data/
-          datalad get -r ./neuroscope/
-          datalad get -r ./openephysbinary/v0.4.4.1_with_video_tracking/
-          datalad get -r ./blackrock/
-          datalad get -r ./intan/
-          datalad get -r ./spikegadgets/
-          datalad get -r ./spike2/
-          datalad get -r ./spikeglx/Noise4Sam_g0/Noise4Sam_g0_imec0/
-          datalad get -r ./spikeglx/TEST_20210920_0_g0/
-          datalad get -r ./phy/phy_example_0/
-          datalad get -r ./axona/
-          datalad get -r ./cellexplorer/
-          datalad get -r ./axon/
-          datalad get -r ./edf/
-          cd ..
-      - if: steps.cache-ophys-datasets.outputs.cache-hit == false && (matrix.os == 'ubuntu-latest' || matrix.os == 'windows-latest')
-        name: "Force GIN: ophys download"
-        run: datalad install -rg https://gin.g-node.org/CatalystNeuro/ophys_testing_data
-      - if: steps.cache-behavior-datasets.outputs.cache-hit == false && (matrix.os == 'ubuntu-latest' || matrix.os == 'windows-latest')
-        name: "Force GIN: behavior download"
-        run: datalad install -rg https://gin.g-node.org/CatalystNeuro/behavior_testing_data
+          pip install awscli==1.25.27
+          aws configure set aws_access_key_id ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws configure set aws_secret_access_key ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
```

Comment:
Shouldn't the region be configured as well?

Reply:
No need apparently

Reply:
Great. How does it do it? Maybe that's the default one? Maybe it is because of where the machines that execute are? Maybe it gets the default location from the account associated with the secrets.

Reply:
Dunno. If I had to guess I'd say it checks the credentials for all existing S3 buckets and only uses the region if there are multiple buckets of the same name?

Reply:
I see, I was somehow thinking that you that transaction costs or speeds would be different if you specify the wrong region.

Reply:
That's definitely something to always be aware of when deploying EC2 or related instances through AWS - using a different region from where your S3 bucket is will generate spurious costs (and this bucket does exist on us-east-2, same as DANDI).